package.json

```diff
@@ -8,13 +8,19 @@
   "types": "dist/index.d.ts",
   "main": "dist/haws.umd.js",
   "module": "dist/index.js",
+  "exports": {
+    "node": {
```

Comment:
Do we have to make this exclusive to node?

Reply:
I tried to take the lightest touch solution, i.e., only modify for node.